content/blog/automation-api/index.md

```diff
@@ -0,0 +1,521 @@
+---
+title: "The Pulumi Automation API - The Next Quantum Leap in IaC"
+date: 2020-10-19
+meta_desc: "Introducing the Pulumi Automation API - Scaling Cloud Projects with Software, Not Humans"
+meta_image: automation_api.png
+authors:
+    - evan-boyle
+tags:
+    - automation api
+---
+
+Today’s Infrastructure as Code platforms let organizations build rich, reliable, and complex cloud-based applications and architectures. But as teams move to modern cloud technologies,  they continue to search for ways to drive increased software-driven automation. Although modern Infrastructure as Code tools bring key software engineering benefits to cloud engineering, they remain focused on human-driven workflows. For example, a person running `pulumi up` at their terminal or wiring the Pulumi CLI into their CI/CD system.
+
+To scale up how we deploy and manage cloud infrastructure and ultimately unlock the cloud's value and agility, we believe it will be critical to building software systems around our Infrastructure as Code platforms - systems that scale with software, not just humans.
+
+<!--more-->
+
+Pulumi's approach of using general purpose languages enables Infrastructure as Software -- bringing all of the great lessons learned over decades of building software to your infrastructure (languages, IDEs, testing, debugging, componentization, packaging, versioning, and more).
```

Comment:
This paragraph feels a little oddly out of place to me.  Not a big deal - but doesn't seem to connect the general intro above to the specific announcement below.

Reply:
I got feedback that it was important to explicitly define "Infra as Software", which is where this came from.